java/vector/src/test/java/org/apache/arrow/vector/ipc/TestArrowReaderWriter.java

```diff
@@ -751,4 +754,43 @@ public void testChannelReadFullyEos() throws IOException {
       assertEquals(10, arrBuf.getInt(0));
     }
   }
+
+  @Test
+  public void testCustomMetaData() throws IOException {
+
+    VarCharVector vector = newVarCharVector("varchar1", allocator);
+    vector.allocateNewSafe();
+    ValueVectorDataPopulator.setVector(vector, "foo", "bar", "baz");
```

Comment:
are any values needed?  If so i imagine just one is needed?

Reply:
ok, no special reason, made it look like a normal file data. Removed now.